website/docs/getting-started/setup/installation-guides/google-cloud-run/manage-traffic.mdx

```diff
@@ -0,0 +1,98 @@
+* If you want scalable, secure, and highly available services, enable [Memorystore for Redis API](https://console.cloud.google.com/apis/dashboard).
+* Ensure that you have created a manual backup for your instance.
+
+## Create a Redis instance
+
+This guide is optional, and you can skip to [Create Service](#create-service). Follow the steps below if you want scalable, secure, and highly available services. Ensure you have enabled the **Memorystore for Redis** API as mentioned in step 5 of the [Before you begin](#before-you-begin) section.
+
+1. Go to the [Memorystore for Redis](https://console.cloud.google.com/memorystore/redis/instances) page in the Google Cloud console.
+
+2. Click **Create Instance**.
+    
+3. On the **Create a Redis instance** page, enter the configurations as shown below:
+
+    * In the **Instance ID** field, set an id. Example: `instanceid`.
+    * In the **Tier Selection** list, select one of the options:
+      * **Basic**: One primary Redis node. No automatic failover for High Availability.
+      * **Standard**: Provides *High Availability* and fails over to replica in case of primary node failure.
+    For more information about tiers, see [Redis Tier Capabilities](https://cloud.google.com/memorystore/docs/redis/redis-tiers).
+    * In the **Capacity** box, enter `10`.
+    * In the **Region** box, select a region for your instance.
+    * In the **Zone** box, select a zone for your instance.
+    * If you selected the **Standard** tier above, in the **Configure read replicas** dropdown list, select **2 Read Replicas** .
+    * Under **Set up connection**, select the **default** option for **Network**.
+    * Expand the **Configuration** section. Select **6.x** option(recommended) for **Version**.
+
+4. Click the **Create** button.
+
+5. Once the Redis instance is ready, note down the **Primary Endpoint** to use later.
+
+<figure>
+    <img src="/img/redis-primary-endpoint.png" style= {{width:"100%", height:"auto"}} alt="Primary Endpoint of the Redis Instance"/>
+    <figcaption align = "center"><i>Primary Endpoint of the Redis Instance</i></figcaption>
+</figure>
+
+## Change Appsmith installation
+
+* Edit the Appsmith instance, and go to the **Autoscaling** section.
+* Set **Maximum number of instances** to `4`.
+* Under **Environment variables**, click **Add Variable** to add below variable in the **Name** and **Value** text boxes as shown below: <br/>
+
+| Name | Value |
+|------------------------|------------------------------------------| 
+| **APPSMITH_REDIS_URL** | If you followed the steps in the [Create Redis Instance](#create-a-redis-instance) section, add the host and port from the Primary Endpoint of the Redis instance you noted down in step 5. <br/><br/> <i>Format</i>: <br/>`redis://<HOST>:<PORT>`|
+
+* ---Add steps to update the installation ---- TO DO.
```

Comment:
@pratapaprasanna - Here the steps are needed for updating or redeploying the installation.